dotenv/mod.ts

```diff
@@ -221,13 +221,6 @@ export interface LoadOptions {
    * @default {"./.env.defaults"}
    */
   defaultsPath?: string | null;
-
-  /**
-   * List of Env variables to read from process. By default, the complete Env is
-   * looked up. This allows to permit access to only specific Env variables with
-   * `--allow-env=ENV_VAR_NAME`.
-   */
-  restrictEnvAccessTo?: StringList;
```

Comment:
Let's keep this option only in typing in a few versions to prevent immediately breaking users code (in type checks)